File: CRM/src/app/shared/eHospital_DongNai_A/CSKH_The_KhuyenMaiNguoiThan.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CSKH_The_KhuyenMaiNguoiThan } from './CSKH_The_KhuyenMaiNguoiThan.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CSKH_The_KhuyenMaiNguoiThanService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'The_QD_KhuyenMaiNguoiThan_Id', 'The_Id', 'LoaiKhachHang_Id', 'DichVu_Id', 'TyLe', 'SoTien', 'TatCaDichVu', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: CSKH_The_KhuyenMaiNguoiThan[] | undefined;
    ListFilter: CSKH_The_KhuyenMaiNguoiThan[] | undefined;
    FormData!: CSKH_The_KhuyenMaiNguoiThan;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CSKH_The_KhuyenMaiNguoiThan";
    }
}

File: Service_eHospital_DongNai_A/Implement/CSKH_The_KhuyenMaiNguoiThanService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CSKH_The_KhuyenMaiNguoiThanService : BaseService<CSKH_The_KhuyenMaiNguoiThan, ICSKH_The_KhuyenMaiNguoiThanRepository>
    , ICSKH_The_KhuyenMaiNguoiThanService
    {
    private readonly ICSKH_The_KhuyenMaiNguoiThanRepository _CSKH_The_KhuyenMaiNguoiThanRepository;
    public CSKH_The_KhuyenMaiNguoiThanService(ICSKH_The_KhuyenMaiNguoiThanRepository CSKH_The_KhuyenMaiNguoiThanRepository) : base(CSKH_The_KhuyenMaiNguoiThanRepository)
    {
    _CSKH_The_KhuyenMaiNguoiThanRepository = CSKH_The_KhuyenMaiNguoiThanRepository;
    }
    }
    }

